ui/components/app/permissions-connect-header/index.scss

```diff
@@ -33,24 +19,13 @@
     margin-top: 16px;
   }
 
-  &__text,
   &__subtitle {
     @include H6;
 
     text-align: center;
     color: $Grey-500;
   }
 
-  &__text {
-    width: 100%;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    margin-top: 8px;
-
-    /*rtl:ignore*/
-    direction: rtl;
```

Comment:
This was used to ensure that long domain names were ellipses on the right, rather than the left. This was to prevent phishers from mimicking the domains of legitimate sites.

If we let the ellipsis stay on the right, then the domains `legitimatelongdomainname.com` and `legitimatelongdomainname.malicious.com` might both look the same (e.g. like `legitimatelongdomainname...`).

Putting the ellipses on the _left_ preserves the most important part of the domain, which is the end.

Reply:
It looks like the new site origin component doesn't handle long domains very well right now. They just expand the component to exceed the width of the screen. e.g. try with `http://llanfairpwllgwyngyllgogerychwyrndrobwllllantysiliogogogoch.co.uk/` or `https://3.141592653589793238462643383279502884197169399375105820974944592.eu/`